async/README.md

```diff
@@ -130,3 +130,16 @@ const [branch1, branch2] = tee(gen());
   }
 })();
 ```
+
+## timeout
+
+Create a promise which will be rejected with `TimeoutError` when a given delay
+is exceeded.
+
+```typescript
+import { timeout } from "https://deno.land/std/async/mod.ts";
+
+const delayedPromise = delay(1000);
+const result = await timeout(delayedPromise, 10);
+// Above throws `TimeoutError` after 10 ms
```

Comment:
Thanks for updating the docs :+1: 

This comment might be better placed above the related line rather than below, as would be the case in source code anyway.

Reply:
OK. I'll fix it.